source/develop/troubleshooting-nfs-storage-issues.md

```diff
@@ -147,6 +147,37 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
       # systemctl start rpcbind 
       # systemctl start nfs-server
 
+#### Synology DSM 7
+Synology DSM 7 adds / extends Access Control Lists (ACLs) on all shared folders, and removes `groupadd` and `useradd`...  
+To allow ovirt nodes to connect via NFS the ACL permissions and /etc/exports file need to be updated manually (*take backups before editing!*).
+
+1. Create the shared folder required as usual and allow you oVirt hosts IP's in the NFS permissions section
+
+2. Login to you Synology NAS via SSH (search the web for 'synology enable ssh access') as root or sudo to root once logged in
+
+3. Carefully add kvm (id 36) to /etc/group and vdsm (id 36) to /etc/passwd and /etc/shadow, if you need an encrypted password the python command below will create one
+
+        # python3 -c 'import crypt,getpass;pw=getpass.getpass();print(crypt.crypt(pw) if (pw==getpass.getpass("Confirm: ")) else exit())'
+
+4. Update the Synology ACLs to allow access to kvm:vdsM
+
+        # EXPORT_DIR=/volumeX/...
+
+        # synoacltool -set-owner "$EXPORT_DIR" group kvm:allow:rwxpdDaARWcCo:fd--
+        # synoacltool -add "$EXPORT_DIR" user:vdsm:allow:rwxpdDaARWcCo:fd--
+        # synoacltool -add "$EXPORT_DIR" group:kvm:allow:rwxpdDaARWcCo:fd--
```

Comment:
From oVirt point of view, having 36:36 is good enough, there is no need
to add user and group. Does synoacltool support uid/guid or it requires user/group names?

Reply:
Not sure I'll have a look, my setup was 'upgraded' and I'd already added the user and group with vanilla Linux commands on an earlier version.

Reply:
Unfortunately yes, it must be doing a lookup...

```
synoacltool -set-owner "$EXPORT_DIR" group 36:allow:rwxpdDaARWcCo:fd--
(synoacltool.c, 790)No such Group
```